tests/printer/comments/expected/jsx.res.txt

```diff
@@ -14,9 +14,9 @@ module Cite = {
 
 <A /* comment */ />
 
-<A
-// Comment
-/>
+<A>
```

Comment:
Could we keep tags opening if these are comments inside? @cknitt 
I found this change was introduced in https://github.com/rescript-lang/syntax/commit/e913c4cf9881efa8d91778e72cc50a61ce651b78

Reply:
Fine with me! That would make it easier to add code after the comment anyway.